core/src/main/kotlin/parsers/MarkdownParser.kt

```diff
@@ -77,34 +79,63 @@ class MarkdownParser (
                 params =
                 if (node.type == MarkdownElementTypes.ORDERED_LIST) {
                     val listNumberNode = node.children.first().children.first()
-                    mapOf("start" to text.substring(listNumberNode.startOffset, listNumberNode.endOffset).dropLast(2))
+                    mapOf("start" to text.substring(listNumberNode.startOffset, listNumberNode.endOffset).trim().dropLast(1))
                 } else
                     emptyMap()
             )
         }
 
-        private fun linksHandler(node: ASTNode): DocTag {
-            val linkNode = node.children.find { it.type == MarkdownElementTypes.LINK_LABEL }!!
-            val link = text.substring(linkNode.startOffset + 1, linkNode.endOffset - 1)
-
-            val dri: DRI? = if (link.startsWith("http") || link.startsWith("www")) {
-                null
-            } else {
-                    resolveKDocLink(
-                        resolutionFacade.resolveSession.bindingContext,
-                        resolutionFacade,
-                        declarationDescriptor,
-                        null,
-                        link.split('.')
-                    ).also { if (it.size > 1) DokkaConsoleLogger.warn("Markdown link resolved more than one element: $it") }.firstOrNull()//.single()
-                        ?.let { DRI.from(it) }
+        private fun resolveDRI(link: String): DRI? = if (link.startsWith("http") || link.startsWith("www")) {
```

Comment:
This should be done in another method I think, as the list of URI prefixes can be longer (`file`, `ftp`, `mailto` and so on). Moreover I think we could recognise it somehow (maybe by a colon after the first word or by creating a `java.net.URL` and catching `MalformedURLException`)

Reply:
Second option seems nice

Reply:
Done